### PR TITLE
[mrg+1] Showcase example: Bézier curves & SVG 

### PR DESCRIPTION
**Pull request by rougier**
This is an example showing Bézier curves using a simple SVG parser on a simple path (firefox logo).
This aims at illustrating that matplotlib can also be used as a regular drawing library.

firefox

### CODE REVIEW

File: examples/showcase/firefox.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import re
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.path import Path
+import matplotlib.patches as patches
+
+# From: http://raphaeljs.com/icons/#firefox
```
Comment (WeatherGod) [inline examples/showcase/firefox.py]: This address redirects to the main page. And clicking on "Free Icons" gets me a 404 error. I see no information about licensing. I am almost certain that firefox icons from mozilla aren't compatible with BSD-like licenses (which is why the BSD OSes ship with rebranded Firefox).

Comment (rougier) [inline examples/showcase/firefox.py]: Thanks, I'll check.

Comment (rougier) [inline examples/showcase/firefox.py]: I only found this where they said icons were under MIT license: https://github.com/Sciss/RaphaelIcons

**Comment by tacaswell** [comment]
This is clearly not the _actual_ firefox logo https://www.mozilla.org/en-US/styleguide/identity/firefox/branding/  but a 'iconified' version and I think we can trust the source which is MIT licensed.

If anyone challenges us on this, we roll over immediately. 

This falls into the same category as http://matplotlib.org/examples/pylab_examples/dolphin.html

**Comment by NelleV** [comment]
Thanks for the patch, Nicolas!

**Comment by tacaswell** [comment]
backported to v2.x as c91efe1

File: examples/showcase/firefox.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
...61-3.423-0.248c0.013,0.015,3.384,0.589,3.981,1.411c0,0-1.431,0-2.856,0.41c-0.065,0.019,5.242,0.663,6.327,5.966c0,0-0.582-1.213-1.301-1.42c0.473,1.439,0.351,4.17-0.1,5.528c-0.058,0.174-0.118-0.755-1.004-1.155c0.284,2.037-0.018,5.268-1.432,6.158c-0.109,0.07,0.887-3.189,0.201-1.93c-4.093,6.276-8.959,2.539-10.934,1.208c1.585,0.388,3.267,0.108,4.242-0.559c0.982-0.672,1.564-1.162,2.087-1.047c0.522,0.117,0.87-0.407,0.464-0.872c-0.405-0.466-1.392-1.105-2.725-0.757c-0.94,0.247-2.107,1.287-3.886,0.233c-1.518-0.899-1.507-1.63-1.507-2.095c0-0.366,0.257-0.88,0.734-1.028c0.58,0.062,1.044,0.214,1.537,0.466c0.005-0.135,0.006-0.315-0.001-0.519c0.039-0.077,0.015-0.311-0.047-0.596c-0.036-0.287-0.097-0.582-0.19-0.851c0.01-0.002,0.017-0.007,0.021-0.021c0.076-0.344,2.147-1.544,2.299-1.659c0.153-0.114,0.55-0.378,0.506-1.183c-0.015-0.265-0.058-0.294-2.232-0.286c-0.917,0.003-1.425-0.894-1.589-1.245c0.222-1.231,0.863-2.11,1.919-2.704c0.02-0.011,0.015-0.021-0.008-0.027c0.219-0.127-2.524-0.006-3.76,1.604C9.674,8.045,9.219,7.95,8.71,7.95c-0.638,0-1.139,0.07-1.603,0.187c-0.05,0.013-0.122,0.011-0.208-0.001C6.769,8.04,6.575,7.88,6.365,7.672c0.161-0.18,0.324-0.356,0.495-0.526C9.201,4.804,12.43,3.357,16.002,3.356z"
```
Comment (QuLogic) [inline examples/showcase/firefox.py]: Are examples not run through pycodestyle? This is definitely too long.

Comment (tacaswell) [inline examples/showcase/firefox.py]: apparently not.  

I think this ok to leave as a long string as it is input data and not meant to be human readable/editable anyway.  Either your editor will soft-wrap it (in which case it takes up the same visual space as if we had wrapped it by hand) or your editor truncates it, and then it takes up only one line and there was much rejoicing ;)

**Pull request merged**